docusaurus/docs/iOS/02-tutorials/03-livestream.mdx

```diff
@@ -5,328 +5,325 @@ description: How to build a livestream experience using Stream's video SDKs
+
+The livestream layout is built using standard SwiftUI. The [VideoRenderer](../04-ui-components/02-video-renderer.mdx) component is provided by Stream.
+**VideoRenderer** renders the video and a fallback. You can use it for rendering the local and remote video.
+
+### Step 4 - (Optional) Publishing RTMP using OBS
+
+The example above showed how to publish your phone's camera to the livestream.
+Almost all livestream software and hardware supports RTMPS.
+[OBS](https://obsproject.com/) is one of the most popular livestreaming software packages and we'll use it to explain how to import RTMPS.
+So let's see how to publish using RTMPs. Feel free to skip this step if you don't need to use RTMPs.
+
+A. Log the URL & Stream Key
 
 ```swift
-VideoCallParticipantView(
-    participant: first,
-    availableSize: reader.size,
-    contentMode: .scaleAspectFit,
-    edgesIgnoringSafeArea: .bottom
-) { participant, view in
-    view.handleViewRendering(for: participant) { size, participant in }
+if let rtmp = state.ingress?.rtmp {
+    print("RTMP url and streamingKey \(rtmp)")
 }
 ```
 
-We are listening for changes to the broadcasting state of the call by setting up a subscription task on `call`s event `AsyncStream`. We are interested only for the events that relate to broadcasting (`typeCallBroadcastingStartedEvent` & `typeCallBroadcastingStoppedEvent`).
+For example, you can listen to this value with the `onReceive` modifier:
 
 ```swift
-.task {
-    for await videoEvent in call.subscribe() {
-        switch videoEvent {
-        case .typeCallBroadcastingStartedEvent:
-            self.isBroadcasting = true
-            self.isLoading = false
-        case .typeCallBroadcastingStoppedEvent:
-            self.isBroadcasting = false
-            self.isLoading = false
-        default:
-            break
-        }
+.onReceive(state.$ingress) { _ in
+    if let rtmp = state.ingress?.rtmp {
+        print("RTMP url and streamingKey \(rtmp)")
     }
 }
 ```
 
-When the call is started, there's still no livestream available. We need to explicitly start it, by calling the `startBroadcasting` method from the `call` object:
+B. Open OBS and go to settings -> stream
 
-```swift
-try await call.startBroadcasting()
-```
+- Select "custom" service
+- Server: equal to the server URL from the log
+- Stream key: equal to the stream key from the log
 
-When this method is called, it takes few seconds until the broadcasting is started. You can show a loading spinner until you receive the `.typeCallBroadcastingStartedEvent` event.
+Press start streaming in OBS. The RTMP stream will now show up in your call just like a regular video participant.
+Now that we've learned to publish using WebRTC or RTMP let's talk about watching the livestream.
 
-When the call is being broadcasted, we show a "Stop stream" button to the hosts, to allow them to stop broadcasting:
+### Step 5 - Viewing a livestream (WebRTC)
```

Comment:
```suggestion
### Step 6 - Viewing a livestream (WebRTC)
```

Reply:
Shall we add the code that is required for the viewer?

Reply:
skipping to keep the same structure with Android tutorial